src/main/java/lotto/controller/LottoController.java

```diff
@@ -0,0 +1,88 @@
+package lotto.controller;
+
+import java.util.List;
+import lotto.domain.amount.PurchaseAmount;
+import lotto.domain.lotto.BonusNumber;
+import lotto.domain.lotto.LottoMachine;
+import lotto.domain.lotto.LottoStatistic;
+import lotto.domain.lotto.Lottos;
+import lotto.domain.lotto.WinningLotto;
+import lotto.dto.LottosResponse;
+import lotto.view.InputView;
+import lotto.view.OutputView;
+
+public class LottoController {
+
+    private final LottoMachine lottoMachine;
+    private final LottoStatistic lottoStatistic;
+
+    public LottoController() {
+        this.lottoMachine = LottoMachine.initial();
+        this.lottoStatistic = LottoStatistic.initial();
+    }
+
+    public void run() {
+        PurchaseAmount purchaseAmount = receivePurchaseAmount();
+        Lottos lottos = purchaseLottos(purchaseAmount);
+        WinningLotto winningLotto = receiveWinningLotto();
+        computeStatistics(lottos, winningLotto, purchaseAmount);
```

Comment:
computeStatistics라는 메서드는 흐름을 제어한다는 Controller의 취지에는 맞지않지않을까요?
LottoStatistic 도메인 객체를 제어할 수 있는 새로운 클래스를 만들고 메시지를 보내는 것은 어떻게 생각하시나요?
```suggestion
        private StatisticProducer statisticProducer;
        statisticProducer.compute(lottos, winningLotto, purchaseAmount)
```